### PR TITLE
MAT-7279 updated measure type mappings

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR Template

Jira Ticket: [MAT-7279](https://jira.cms.gov/browse/MAT-7279)
(Optional) Related Tickets:

### Summary
updated measure type mappings

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e. Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/BaseConfigurationTypes.java [hunk 1]
```diff
@@ -9,6 +9,8 @@ public enum BaseConfigurationTypes {
   APPROPRIATE_USE_PROCESS("Appropriate Use Process"),
   @JsonProperty("Cost/Resource Use")
   COST_OR_RESOURCE_USE("Cost/Resource Use"),
+  @JsonProperty("Resource Use")
+  RESOURCE_USE("Resource Use"),
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/BaseConfigurationTypes.java:11]: I think line 10 & 11 need to be replaced by this.

File: src/main/java/gov/cms/madie/models/measure/BaseConfigurationTypes.java [hunk 2]
```diff
@@ -17,6 +19,8 @@ public enum BaseConfigurationTypes {
   OUTCOME("Outcome"),
   @JsonProperty("Patient Engagement/Experience")
   PATIENT_ENGAGEMENT_OR_EXPERIENCE("Patient Engagement/Experience"),
+  @JsonProperty("Experience")
+  EXPERIENCE("Experience"),
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/BaseConfigurationTypes.java:19]: and line 20 and 21 by this

**Review by adongare — APPROVED** [review]
